README.md

```diff
@@ -232,6 +232,16 @@ instance_groups:
               replicas: 2
 ```
 
+## CF create-service configuration parameters
+App developers can customize the following parameters. See the [Redis documentation](https://redis.io/topics/config) for more detail. In particular, look at the self documented redis.conf file for your deployed Redis version.
+
+| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Property&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Options | Description |
```

Comment:
Please remove the nbsp's; it's only readable in a web browser.

Reply:
Looks crappy in the browser then.   Looking for another solution.

Reply:
It displays crappy with default settings.     I removed the nbsp characters and added some strategic line breaks.  It makes it less noisy in the file and still get a better looking HTML output.